### DISCUSSION TITLE
Where are "West EU" and "Central EU" when creating a new project?

### DISCUSSION THREAD

**Discussion by ellis** [supabase/supabase#441] [Questions] [answered]
I'm trying out Supabase and creating a new project.  It gives me region options for "West EU" and "Central EU" (among others, of course).  The terminology isn't standard, so I'm not sure what those mean.  For example, is "West EU" Spain, which is about as far west as you can get, yet it's in the Central European time zone; or is it perhaps England, which is less west, but in a more westerly time zone and soon to leave the EU?  Clarification appreciated :)

**Comment by kiwicopple** [comment, marked as answer]
Hey @ellis - it's non-standard because we started on another cloud provider before switching to AWS. Here are the region mappings:

```
EAST_US: 'us-east-1', // North Virginia
WEST_US: 'us-west-1', // North California
SOUTHEAST_ASIA: 'ap-southeast-1', // Singapore
WEST_EU: 'eu-west-1', // Ireland
CENTRAL_EU: 'eu-central-1', // Frankfurt
CENTRAL_CANADA: 'ca-central-1', // Central Canada
SOUTH_ASIA: 'ap-south-1', // Mumbai
SOUTH_AMERICA: 'sa-east-1', // Sao Paulo
```

**Reply by ellis** [reply]
Thanks, much appreciated
